ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -5159,10 +5158,7 @@ let vm_cd_insert printer rpc session_id params =
 let host_careful_op op warnings fd _printer rpc session_id params =
   let uuid = List.assoc "uuid" params in
   let host = Client.Host.get_by_uuid ~rpc ~session_id ~uuid in
-  let pool = List.hd (Client.Pool.get_all ~rpc ~session_id) in
-  let _ (* unused variable 'pool_master' *) =
-    Client.Pool.get_master ~rpc ~session_id ~self:pool
-  in
+  let _ (* unused variable 'pool_master' *) = get_master ~rpc ~session_id in
```

Comment:
Why do we need this?

Reply:
It's old code, I just refactored it. I removed `List.hd` and replaced `Client.Pool.get_master` with Cli_util.get_master`. It seems that the returned `pool_master` has never been used (it's used before). I don't know why it's commented out instead of removed. 
Can I remove these unused codes?

Reply:
Then you can also remove the related, commented-out line below. This sort of thing is best done in a separate commit, with a commit message explaining it (so we don't need to ask in the code review).

Reply:
Removed and created a new commit.